### PR TITLE
[LexikJWTAuthenticationBundle] Add aliases

### PR DESCRIPTION
**Pull request by ogizanagi**
ping @chalasr

### CODE REVIEW

File: lexik/jwt-authentication-bundle/2.3/manifest.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "copy-from-recipe": {
         "etc/": "%ETC_DIR%"
     },
+    "aliases": ["jwt-auth"],
```
Comment (javiereguiluz) [inline lexik/jwt-authentication-bundle/2.3/manifest.json]: If it's decided that this bundle is "the JWT solution for Symfony", then we could even go for adding `jwt` too as an alias.

Comment (ogizanagi) [inline lexik/jwt-authentication-bundle/2.3/manifest.json]: I wanted at first, but the bundle is a solution for setting a jwt authentication, not for consuming external jwt tokens for instance (even if it requires libraries that will probably do natively).

Comment (chalasr) [inline lexik/jwt-authentication-bundle/2.3/manifest.json]: Indeed, the bundle is written on top of libraries like [lcobucci/jwt](https://github.com/lcobucci/jwt), it is only about authentication while jwt can be used in different contexts. 👍 for jwt-auth

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**